test/simple.spec.ts

```diff
@@ -430,7 +432,7 @@ enum Episode {
 \\"\\"\\"A connection to a list of items.\\"\\"\\"
 type CharacterConnection {
   \\"\\"\\"Information to aid in pagination.\\"\\"\\"
-  pageInfo: PageInfo!
+  pageInfo: PageInfo
```

Comment:
https://relay.dev/graphql/connections.htm#sec-Connection-Types.Fields.PageInfo

>  “Connection Type” must contain a field called pageInfo. This field must return a non-null PageInfo object, as defined in the “PageInfo” section below.

Reply:
You're a tad hypocritical here mate. How can you drop the spec on me here and ignore it on the other moment?